src/geotiff-validator/README.md

```diff
@@ -0,0 +1,41 @@
+# GeoTIFF Validator
+
+## Configuration
+
+Provide a `config.yaml` with the follwing keys and adapt it to your needs.
+
+| key                 | mandatory | type     | default                          |
```

Comment:
we only provide default values in the config and not in the code right?
I would expect that the config come into play when there is no value set in the config file